DQM/SiStripMonitorClient/test/mergeBadChannel_Template_cfg.py

```diff
@@ -0,0 +1,86 @@
+import FWCore.ParameterSet.Config as cms
+
+import FWCore.ParameterSet.VarParsing as VarParsing
+
+process = cms.Process("BadChannelMerge")
+
+#prepare options
+
+options = VarParsing.VarParsing("analysis")
+
+options.register ('globalTag',
+                                    "DONOTEXIST::All",
+                                    VarParsing.VarParsing.multiplicity.singleton, # singleton or list
+                                    VarParsing.VarParsing.varType.string,          # string, int, or float
+                                    "GlobalTag")
+options.register ('dqmFile',
+                                    "",
+                                    VarParsing.VarParsing.multiplicity.singleton, # singleton or list
+                                    VarParsing.VarParsing.varType.string,          # string, int, or float
+                                    "DQM root file")
+options.register ('runNumber',
+                                    1,
+                                    VarParsing.VarParsing.multiplicity.singleton, # singleton or list
+                                    VarParsing.VarParsing.varType.int,          # string, int, or float
+                                    "run number")
+options.parseArguments()
+
+
+process.MessageLogger = cms.Service("MessageLogger",
+    destinations = cms.untracked.vstring('cout','cerr','MergedBadComponents'), #Reader, cout
+    categories = cms.untracked.vstring('SiStripQualityStatistics'),
+
+    debugModules = cms.untracked.vstring('siStripDigis', 
+                                         'siStripClusters', 
+                                         'siStripZeroSuppression', 
+                                         'SiStripClusterizer',
+                                         'siStripOfflineAnalyser'),
+    cerr = cms.untracked.PSet(threshold = cms.untracked.string('ERROR')
+                              ),
+    cout = cms.untracked.PSet(threshold = cms.untracked.string('INFO'),
+                                default = cms.untracked.PSet(limit=cms.untracked.int32(0))
+                              ),
+    MergedBadComponents = cms.untracked.PSet(threshold = cms.untracked.string('INFO'),
+                                default = cms.untracked.PSet(limit=cms.untracked.int32(0)),
+                                SiStripQualityStatistics = cms.untracked.PSet(limit=cms.untracked.int32(100000))
+                                )
+                                    
+)
+process.load('Configuration.Geometry.GeometryIdeal_cff')
+process.load("Configuration.Geometry.GeometryReco_cff")
+
+process.load('Configuration.StandardSequences.FrontierConditions_GlobalTag_condDBv2_cff')
+from Configuration.AlCa.GlobalTag_condDBv2 import GlobalTag
+process.GlobalTag = GlobalTag(process.GlobalTag, options.globalTag, '')
+
```

Comment:
I also would think that 
+process.load('Configuration.Geometry.GeometryIdeal_cff')
+process.load("Configuration.Geometry.GeometryReco_cff")

should be replaced by 
process.load("Configuration.StandardSequences.GeometryDB_cff")